metadata.rb

```diff
@@ -7,3 +7,5 @@
 version '0.1.4'
```

Comment:
You'll need to bump the version as approval of this PR will automatically ship to supermarket.chef.io (be sure to rebase on master also so you know what to bump to).